packages/launch/src/launch/index.tsx

```diff
@@ -35,6 +35,9 @@ const FocusedLaunchModal: React.FunctionComponent< Props > = ( {
 	const isModalTitleVisible = useSelect( ( select ) =>
 		select( LAUNCH_STORE ).isModalTitleVisible()
 	);
+	const shouldDisplaySuccessView = useSelect( ( select ) =>
+		select( LAUNCH_STORE ).shouldDisplaySuccessView()
+	);
```

Comment:
More of a curiosity of mine, but I wonder if it'd be more efficient to `select` from the Launch Store only once:

```ts
const { isModalDismissible, isModalTitleVisible, shouldDisplaySuccessView } = useSelect(
	( select ) => {
		const launchSelect = select( LAUNCH_STORE );
		return {
			isModalDismissible: launchSelect.isModalDismissible(),
			isModalTitleVisible: launchSelect.isModalTitleVisible(),
			shouldDisplaySuccessView: launchSelect.shouldDisplaySuccessView(),
		};
	}
);
```

Reply:
Ideally I think we should call `useSelect` only once in a component if the dependencies Array (second argument of `useSelect`) is the same and in this case is `undefined`.

Example with deps: https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/gutenboarding/hooks/use-free-domain-suggestion.ts#L15-L26
Example with multiple stores accessed in one call: https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx#L47-L57

Reply:
Added the change in https://github.com/Automattic/wp-calypso/pull/47808/commits/9fc3bc7ecfad225152334f876b0d99f28ae156a5

I'm not going to create a janitorial issue for this but let's keep in mind and use this pattern at least until we have something better from readability & performance PoV.

FYI @Automattic/luna